README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
+don't exist.
+
+Above, when we said that the terminal hands requests and receives commands from
+a genie, a thing that _thinks_ and thing that _can find out things_, we were
+actually talking about "the shell." The _shell_ knows how to ask the operating
+system how big the hard drive is. The _shell_ knows how to ask the operating
+system if the web site `flatironschool.com` can be reached. The _shell_ **does
+not** know how to display "Hello World" in 16 point Monaco font on a black
+background, that's the _terminal_.
+
+A shell is a program hosted _inside of a terminal_ that interacts with the
+operating system on your behalf. It's because of the fact that you can't
+interact with a _shell_ without a _terminal_ that some imprecisely call the
+_terminal_ "the shell."
+
+The shell, since it can think and do things, takes input that you type in. It
+looks at that input and asks the operating system to interpret that request.
+When the operating system comes back with results, the shell then hands back to
+the terminal (for displaying) the result of the command.
+
+When you see a terminal, the shell hosted inside of it it will present to you a
+"command prompt." This means, from the computer's perspective "I'm
+listening..." Prompts may look different from computer to computer. Customized
+prompts express developer uniqueness. We've seen weather reports, beautiful
+colors, computer temperatures in prompts. At heart, though, they they all have
+the same function: to carry your command to the operating system and run it.
+
+Here's a terminal application (iTerm2) running a shell (this one happens to be
+called zsh, there are many, it's no big deal).
+
+<img alt="MacOSX Shell" src="https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/osx_shell.png" width=595>
+
+You can ask the computer to do many types of tasks via the shell (via the
+terminal). From the shell's command prompt (that you see in the terminal) you
+can move or rename files just like you would from a window environment.  You
 can _also_ get diagnostic information ("How is my computer?") or help your
 computer recover without requiring a reboot ("Hm, computer, time to kill that
 hung Chrome tab with the Twitch stream...").  You can ask the computer to run
-programs like `ruby my_awesome_program.rb` or even to launch an application!
-
-## Recognize a Shell Interface
+your own programs like `ruby my_awesome_program.rb` or even to launch an
+application!
 
-A shell interface looks like the following pictures below.
+When interacting with a shell via a terminal application, you'll be engaged in
+a cycle called the "prompt-evaluation-response cycle." By this, we mean that
+you will be given a command prompt, you will provide a command that will be
+evaluated by the shell with the help of the operating system and results will
+be printed for you. To trigger evaluation of your command you strike the
+`return` or `enter` keys.
 
```

Comment:
Not sure whether or not this might be easier to understand with concrete examples?